src/policy/marksweepspace/malloc_ms/metadata.rs

```diff
@@ -99,16 +99,20 @@ fn map_active_chunk_metadata(chunk_start: Address) {
         chunk_start + (size / 2)
     );
 
-    assert!(
-        CHUNK_METADATA.try_map_metadata_space(start, size).is_ok(),
-        "failed to mmap meta memory"
-    );
+    CHUNK_METADATA
+        .try_map_metadata_space(start, size, space_name)
```

Comment:
This is good refactoring with clear improvement. The old code put the actual action performed in the `assert!` which is bad.